tests/unit/Services/RetrieveMetadataTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
+ */
+
+namespace OCA\MediaMetadata\Services;
+
+use OCA\MediaMetadata\AppInfo\Application;
+use OCA\MediaMetadata\Tests\MapperTestUtility;
+
+
+/**
+ * Class RetrieveMetadataTest
+ *
+ * @package OCA\MediaMetadata\Services
+ */
+class RetrieveMetadataTest extends MapperTestUtility {
+	/**
+	 * @var \OCA\MediaMetadata\Services\RetrieveMetadata
+	 */
+	protected $retrieveData;
+	/**
+	 * @var \PHPUnit_Framework_MockObject_MockObject|\OCP\IDBConnection
+	 */
+	protected $dbConnection;
+	/**
+	 * @var $entity
+	 */
+	public static $entity = null;
+
+	public static function init() {
```

Comment:
@oparoz Yes, I forgot to remove it. I was trying some other method earlier for which I had written it. I will remove it